src/cfml/system/services/ServerService.cfc

```diff
@@ -1412,11 +1412,11 @@ component accessors="true" singleton {
 
 		args.append( '--web-xml-path' ).append( serverInfo.webXml );
 
-		if( len( serverInfo.webXMLOverrideActual ) ){
-			args.append( '--web-xml-override-path' ).append( serverInfo.webXMLOverrideActual );
+		if( len( serverInfo.webXMLOverride ) ){
+			args.append( '--web-xml-override-path' ).append( serverInfo.webXMLOverride );
```

Comment:
@BalbinoOrtus Why are you changing this variable?  The code doesn't use the override file on purpose, but a copy of it.

Reply:
@bdw429s that variable is the one that is not getting reset and makes the override get stuck between starts, if you want i can add a check for serverInfo.webXMLOverrideActual if serverInfo.webXMLOverride is empty that way i can leave the code as is and reset the variable.

Reply:
added the reset of webXMLOverrideActual @bdw429s